src/backend/distributed/sql/citus--11.1-1--11.2-1.sql

```diff
@@ -9,6 +9,8 @@ DROP FUNCTION pg_catalog.worker_append_table_to_shard(text, text, text, integer)
 #include "udfs/citus_get_transaction_clock/11.2-1.sql"
 #include "udfs/citus_is_clock_after/11.2-1.sql"
 #include "udfs/citus_internal_adjust_local_clock_to_remote/11.2-1.sql"
+#include "udfs/citus_job_list/11.2-1.sql"
+#include "udfs/citus_job_status/11.2-1.sql"
```

Comment:
One small quality of life improvement we'll want is `citus_rebalance_status` which gets the jobid of the currently active rebalance job and returns the status for it.

Reply:
How about not getting any input and checking if the job with highest id is in a non-terminal state?

If in a non-terminal state, we can return `citus_job_status` output, and return 0 rows otherwise.

Reply:
Won't we have concurrent jobs? I'm not sure if I understood it correctly. Looks like there will be a possibility that the job with the highest id is completed but some other job with a lower id is still in progress.

Reply:
jobs yes, rebalance jobs no. There will only ever be one active rebalance job.

Reply:
>Won't we have concurrent jobs?

I guess, not in the near future. Having concurrent tasks, on the other hand may be complete in the next major release.

See https://github.com/citusdata/citus/pull/6576#discussion_r1067201138